src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> convertedTransaction = new ArrayList<>();
+        for (int i = 1; i < data.size(); i++) { // Skip header
```

Comment:
The loop starts at index 1 to skip the header, which is a common practice. Ensure that the input data always contains a header to avoid potential `IndexOutOfBoundsException` if the data list is empty.